Backend-Roadmap/Controllers/UsersController.cs

```diff
@@ -115,6 +115,66 @@ public ActionResult RegisterUser([FromBody] Users newUser)
                 return StatusCode(500, new { error = "An error occurred while registering the user: " + ex.Message });
             }
         }
+        
+
+        [HttpPut]
+        [Route("UpdateUser/{userId}")]
+        public ActionResult UpdateUser(int userId, Users updatedUser)
```

Comment:
Usually it's
```
public ActionResult UpdateUser(int userId, [FromBody] Users userToUpdate)
```
try naming the variables so we know what they mean

Reply:
Try also programming asynchronously with async Task<T>  and  await

Reply:
Updated